create_command.go

```diff
@@ -56,19 +61,32 @@ func (c *CreateCommand) Run(args []string) int {
 		return 1
 	}
 
+	if _, err := create(*c); err != nil {
+		c.Ui.Error(err.Error())
+		return 1
+	}
+
+	return 0
+}
+
+func create(c CreateCommand) (string, error) {
+	snapshotId := ""
+
 	// Check for required command-line args
+	if c.AwsRegion == "" {
+		return snapshotId, errors.New("ERROR: The argument '--region' is required.")
+	}
+
 	if c.InstanceId == "" {
-		c.Ui.Error("ERROR: The argument '--instance' is required.")
-		return 1
+		return snapshotId, errors.New("ERROR: The argument '--instance' is required.")
 	}
```

Comment:
Good call. I wrote this as a golang newb and didn't know better at the time. :-)